tests/test_processing.py

```diff
@@ -169,6 +169,7 @@ def test_nodes_with_none_exclusion(self):
                 'investment_minimum': 0,
                 'label': 'storage',
                 'inflow_conversion_factor': 1,
+                'loss_constant': 0,
```

Comment:
I have no idea what test_processing is testing - so probably thats fine, but I don't now.

Reply:
It tests if the return of the outputlib.processing.param_as_dict() function returns the expected values.